src/app/template-library/template-library.service.ts

```diff
@@ -44,6 +44,10 @@ export class TemplateLibraryService {
     private demo: DemoService,
   ) { }
 
+  static isInCategory(templateType: string, categoryType: string): boolean {
```

Comment:
since we've moved it to the util service, we can use it directly in the component, no need of this function to be in the service file here anymore